### ISSUE TITLE
Enter key submits incomplete record without prompt

### ISSUE THREAD

**Issue by jbest** [zooniverse/notesFromNature#381]
In the Herbarium interface, if you press Enter/Return in some cases, the specimen record is immediately submitted, even with skipped fields, without displaying the "are you sure" prompt that is normally seen when the "finish this record" button is pressed. I haven't found all the cases which cause this behavior, but one I know that can reproduce this is:
enter all fields up to collection number
enter collection number, press Enter
press Enter (with focus on the Month dropdown)
record is submitted

**Comment by denslowm** [comment]
@jbest 
Are you sure if doesn't just take you back to the beginning on the same specimen image?
I just tested it on Firefox (41.0.1) and I was send back to the beginning of the record. It doesn't progress to the next image until I hit Finish.

**Comment by jbest** [comment]
No, it's definitely going to the next record. I'm on Mac OS using Safari 9.0.1. But I tested this in Chrome ( 46.0.2490.71) and got the behavior you described for Firefox (Windows I assume) but in Firefox 41.0.1 for the Mac, I see the same problem that I seen in Safari.

**Comment by denslowm** [comment]
Hmmm. I am on Mac as well (10.10.5). Thanks for doing more testing. I think we might have to revisit this after WeDigBio.
